components/zenFunctions/init_component.sqf

```diff
@@ -45,6 +45,17 @@ if !(isNil 'zen_custom_modules_fnc_register') then
     ["[CA2] Respawners", "Respawn Random Player (INDFOR)", _spawnerIndfor] call zen_custom_modules_fnc_register;
     ["[CA2] Respawners", "Respawn Random Player (Civilian)", _spawnerCiv] call zen_custom_modules_fnc_register;
 
+    if !(isNil 'kyk_ew_fnc_broadcastJammerAdd') then
```

Comment:
Looking good, only downside to this is if that function ever gets renamed for *whatever reason* but that's so unlikely that this is good to go.